pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -37,21 +36,113 @@ If your bot fails to start with a `PrivilegedIntentsRequired` exception, this in
 
 ### Configure the Bot
 You now have both the bot's code and a server to run it on. It's time you to connect the two by changing the bot's configurations.
+This can be done either automatically or manually, and we'll be detailing the steps for both.
+
+#### Automatic configuration
+To make setup much easier, there is a file called `bootstrap_config.py` that represents a script to bootstrap the configuration for you and help you get started immediately
+without having to spend much time copying ids from your newly created server into your configuration file.
+
+**Note**: This phase can be skipped and done manually, but would require extra manual work.
+
+##### 1. Script setup
+##### 1.1. Environment variables
+You will need to create a file called `.env` which will contain two required values for the script to work: `BOT.TOKEN` and `GUILD.ID`
+
+Inside, add the following two lines:
+
+```text
+BOT.TOKEN=YourDiscordBotTokenHere
+GUILD.ID=YourDiscordTestServerIdHere
+```
+See [here](../creating-bot-account) for help with obtaining the bot token and [here](../obtaining-discord-ids#guild-id) for help with obtaining the guild's id
+
+**Note**: The `.env` file will be ignored by commits.
```

Comment:
Either that or we say that they will be ignored **in**  commits.
But I'd say let's go with what you proposed since it's all done by git

How about this ? bfa955417ae582a797c31dd462bb0279fb67c64e